src/features/tree/utils/treeOperations.js

```
@@ -0,0 +1,592 @@
+      }
+      
+      if (node.right) {
+        queue.push(node.right);
+        step(tree, `Added right child ${node.right.value} to queue`, 3, node.value, result);
+      }
+    }
+
+    step(tree, "Level-order traversal completed", 4, null, result);
+    return { result, steps };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)